test/assets/historical.json

```diff
@@ -0,0 +1,34 @@
+{
+  "measures": [
+    {
+      "timestamp": "2000-01-01 00.00",
+      "novaServiceStatus": {"value":"green","description":"desc"},
+      "neutronServiceStatus": {"value":"green","description":"desc"},
+      "cinderServiceStatus": {"value":"green","description":"desc"},
+      "glanceServiceStatus": {"value":"green","description":"desc"},
+      "KPServiceStatus": {"value":"green","description":"desc"},
+      "OverallStatus": {"value":"green","description":"desc"},
+      "FiHealthStatus": {"value":"green","description":"desc"}
+    },
+    {
+      "timestamp": "2000-01-02 00.00",
+      "novaServiceStatus": {"value":"gray","description":"desc"},
+      "neutronServiceStatus": {"value":"gray","description":"desc"},
+      "cinderServiceStatus": {"value":"gray","description":"desc"},
+      "glanceServiceStatus": {"value":"gray","description":"desc"},
+      "KPServiceStatus": {"value":"gray","description":"desc"},
+      "OverallStatus": {"value":"gray","description":"desc"},
+      "FiHealthStatus": {"value":"gray","description":"desc"}
+    },
+    {
+      "timestamp": "2000-01-03 00.00",
+      "novaServiceStatus": {"value":"red","description":"desc"},
+      "neutronServiceStatus": {"value":"red","description":"desc"},
+      "cinderServiceStatus": {"value":"red","description":"desc"},
+      "glanceServiceStatus": {"value":"red","description":"desc"},
+      "KPServiceStatus": {"value":"red","description":"desc"},
+      "OverallStatus": {"value":"red","description":"desc"},
+      "FiHealthStatus": {"value":"red","description":"desc"}
+    }
+  ]
+}
```

Comment:
Just to clarify:
* FiHealth sanity checks results are `OK`, `NOK` and `POK` (see [sanity_tests.js](https://github.com/telefonicaid/fiware-health/blob/master/fiware-region-sanity-tests/resources/parsers/sanity_tests.js)).
* Infographics backend transforms such values, as well as those from the rest of monitoring checks, into "color" values: `green`, `red`, `yellow` and `gray`. Therefore, sample historical data simulates those color values.

Given that the mapping between sanity status <-> colors (or between ODC results <-> colors) was not clear enough, some constants have been added to explicit such mappings.